frontend/src/components/CanteenPublication/ResultsComponents/QualityMeasureResults.vue

```diff
@@ -98,12 +96,21 @@ export default {
     DsfrAccordion,
   },
   data() {
-    const tabs = this.diagnosticSet.map((d) => +d.year)
+    const tabs = this.diagnosticSet.map((d) => ({
+      text: +d.year,
+      value: +d.year,
+      disabled: false,
+    }))
     tabs.sort((a, b) => b - a)
```

Comment:
```suggestion
    tabs.sort((a, b) => b.value - a.value)
```

(j'ai pas testé, devrait être 2023, 2022, 2021 etc non pas 2021, 2022, 2023)

Reply:
Tested, it works. Thanks!